config.js

```diff
@@ -271,6 +271,10 @@ const CONFIG = {
 // CONFIG VALIDATION
 //==============================================================================
 
+if (typeof CONFIG.EMAIL_SUBJECT_PREFIX === 'undefined') {
```

Comment:
Have you thought about using `if (CONFIG.EMAIL_SUBJECT_PREFIX === undefined)` instead?